packages/augur-core/source/contracts/libraries/token/IERC20.sol

```diff
@@ -2,7 +2,8 @@ pragma solidity 0.5.10;
 
 
 contract IERC20 {
-    function totalSupply() public view returns (uint256);
+    uint256 public totalSupply;
```

Comment:
I believe this change is incorrect.  Interfaces should not have members on them (even when they are technically contracts), they should only have functions.  In Solidity it is legal to do the following:
```
contract IERC20 {
    function totalSupply() external view returns (uint256);
}
contract ERC20 is IERC20 {
    uint256 public totalSupply;
}
```
Note: For reasons not entirely clear to me, the interface method needs to be `external`, not `public` for the above to work.

Reply:
It is not legal to do that in Solidity.